.github/workflows/docker-publish.yml

```diff
@@ -39,10 +44,18 @@ jobs:
           tags: |
             type=raw,value=${{ env.RELEASE_VERSION }}
       - uses: docker/build-push-action@v5
+        id: build
         with:
           context: .
           target: mermaid
           push: ${{ github.event_name == 'push' }}
+          platforms: linux/amd64,linux/arm64
           pull: true
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
+      - name: Generate Build Attestation
+        uses: actions/attest-build-provenance@v2
+        with:
+          subject-name: ghcr.io/${{ github.repository }}
+          subject-digest: ${{ steps.build.outputs.digest }}
+          push-to-registry: true
```

Comment:
This step isn't guarded by an if, whereas push *is* guarded by an if.

All things considered, none of this makes sense.